### PR TITLE
Consolidate devel testing

### PR DESCRIPTION
**Pull request by ssbarnea**
Adopts same tox env testing setup as other drivers.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -11,6 +14,8 @@ description =
     Unit testing
 usedevelop = True
 extras = test
+deps =
+    py{36,37,38}-{devel}: git+https://github.com/ansible-community/molecule.git@master#egg=molecule[test]
```
Comment (apatard) [inline tox.ini:18]: missing 39 ?

File: tox.ini [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
 minversion = 3.9.0
-envlist = lint,packaging,py{36,37,38,39},devel,distros
-skipsdist = True
+envlist =
+    lint
+    packaging
+    py{36,37,38,39}
+    py{36,37,38,39}-{devel}
```
Comment (apatard) [inline tox.ini:8]: distros got lost. Replaced by gh actions ?

Comment (ssbarnea) [reply tox.ini:8]: Not really it was never defined, in fact I removed few other unused envs.

File: tox.ini (hunk 1 above)
Comment (ssbarnea) [reply tox.ini]: thanks added.

**Pull request merged**